tutorials/plot_events_and_annotations.py

```diff
@@ -0,0 +1,120 @@
+"""
+The **events** and :class:`Annotations <mne.Annotations>` data structures
+=========================================================================
+
+Events and annotations are quite similar. This tutorial highlights their
+differences and similitudes and tries to shade some light to which one is
+preferred to use in different situations when using MNE.
+Here follows both terms definition from the glossary.
+
+    events
+        Events correspond to specific time points in raw data; e.g., triggers,
+        experimental condition events, etc. MNE represents events with integers
+        that are stored in numpy arrays of shape (n_events, 3). Such arrays are
+        classically obtained from a trigger channel, also referred to as stim
+        channel.
+
+    annotations
+        One annotation is defined by an onset, a duration and a string
```

Comment:
a key difference with the events structure is that the onset here is in seconds whereas in the events structure, it is in samples. Further, in the events structure, it is relative to `raw.first_time` whereas here it is relative to `annot.orig_time`. These differences should be explicitly pointed out in my opinion